docs/MANUAL_CLOUD_TESTS.md

```diff
@@ -0,0 +1,46 @@
+# Ubuntu SRU: Manual cloud test design
```

Comment:
Perhaps move this into sru-scripts (perhaps to replace the README, see my comment on that below)?

Reply:
Done, and renamed the existing README.md to SRU_README.tmpl as it is used by bin/sru_create.py